sections/conclusion/conclusion-chapter.tex

```diff
@@ -1 +1,8 @@
 \section{Conclusion}\label{sec:conclusion}
+Throughout this paper, we have looked into various metrics used for evaluating RS. What we found was that there is no consistent set of metrics used for evaluation, but the most commonly used are NDCG and Recall.
+We found that utilizing bipartite graphs is a popular and intuitive way to handle information in the RS, but in the investigated metrics, graph-based methods are often outperformed by traditional RS such as SVD.
+The cause of this is speculative but may be because methods like SVD make use of explicit rating information, where the investigated graph-based methods use implicit ratings.
+\\\\
+From the knowledge acquired throughout this paper, we propose that instead of only looking into context as information about the interaction, it could be beneficial to expand the model to include side-information about the items or users in addition to the context.
```

Comment:
passer det at vores resultater viser at side information kan være godt?
Man kan måske også nævne at vores eksperiment viser at brug af context ikke umiddelbart gav bedre resultater.
Men det er vel egentlig også gældende for side-information?
Den er sgu lidt svær at konkludere på vores projekt når alt bare er inconclusive 😢